src/components/wizard/wizard.component.ts

```diff
@@ -357,7 +357,7 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
 
         // if there is no validator then return the valid state
         if (!currentStep.validator) {
-            return currentStep.valid;
+            return currentStep._valid;
```

Comment:
I think here is probably good to use the public `valid` property. We largely only want to use the private one inside the component it is a part of. We can use setters/getters in code fine, its mainly in templates we want to avoid them as change detection can end up calling them many many times.